### DISCUSSION TITLE
是否可以提供统一调整行间公式和上下文之间的间距的接口？

### DISCUSSION THREAD

**Discussion by WXiaoyao** [note286/xduts#45] [Q&A] [answered]
在写论文时感觉下图的行间公式和上下文的间距有些略大了，但除了逐个使用\vspace{-0.5em}找不到其他可以统一调整这个间距的方法，看了使用手册也没找到相关的接口。请问有什么办法吗
<img width="441" alt="1676511355952" src="https://user-images.githubusercontent.com/28039355/219242776-32c97bce-7c7b-4eac-80e0-2a667e893700.png">

**Comment by note286** [comment, marked as answer]
学位办要求：

>公式与正文之间一般应空一行。

你可以看一下Microsoft Word模板，也是如此。可能是你这个公式是一行的，所以显着比较单薄，多行公式就不觉得了。

**Reply by WXiaoyao** [reply]
哦哦哦，好的，那没问题了哈哈
